docs/charts/bar.md

```diff
@@ -66,19 +66,19 @@ The bar chart allows a number of properties to be specified for each dataset.
 These are used to set display properties for a specific dataset. For example,
 the color of the bars is generally set this way.
 
-| Name | Type | [Scriptable](../general/options.md#scriptable-options) | [Indexable](../general/options.md#indexable-options) |  Default
+| Name | Type | [Scriptable](../general/options.md#scriptable-options) | [Indexable](../general/options.md#indexable-options) | Default
 | ---- | ---- | :----: | :----: | ----
-| [`backgroundColor`](#styling) | [`Color`](../general/colors.md) | Yes | Yes | `'rgba(0,0,0,0.1)'`
-| [`borderColor`](#styling) | [`Color`](../general/colors.md) | Yes | Yes | `'rgba(0,0,0,0.1)'`
-| [`borderSkipped`](#borderskipped) | `String` | Yes | Yes | `'bottom'`
-| [`borderWidth`](#styling) | `Number` | Yes | Yes | `0`
-| [`data`](#data-structure) | `Object[]` | - | - | **required**
+| [`backgroundColor`](#styling) | [`Color`](../general/colors.md) | Yes | Yes | `'rgba(0, 0, 0, 0.1)'`
```

Comment:
I remember why I didn't use spaces in these strings, it was to prevent line break in the middle of the color value on small screens.

![image](https://user-images.githubusercontent.com/3874900/51893899-956cf580-23a6-11e9-963e-d08fc773eccf.png)

Reply:
We may be able to use a CSS like `table code { white-space: nowrap; }`, but the downside in this case is we also have long text values such as `"'Helvetica Neue', 'Helvetica', 'Arial', sans-serif"` which we would like to wrap.

Reply:
You right, nevermind, let's keep it like that.